applications/usb_uart_bridge/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: USB-UART bridge
+tests:
+  test_build:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf52840_20035
```

Comment:
Could you double check that this is correct? I think it should be `nrf52840_pca20035`

Reply:
Fixed, thanks :)